isis-index.js

```diff
@@ -0,0 +1,45 @@
+const Sequelize = require('sequelize'),
```

Comment:
isis-index was more of a playground, for me to try things without messing up the index.js for everyone, I deleted it